pkg/server/datastore/sqlstore/sqlstore_test.go

```diff
@@ -1676,41 +1665,48 @@ func (s *PluginSuite) TestNodeSelectors() {
 	}
 
 	// assert there are no selectors for foo
-	selectors := s.getNodeSelectors("foo", datastore.TolerateStale)
+	selectors := s.getNodeSelectors("foo", datastore.RequireCurrent)
 	s.Require().Empty(selectors)
-	selectors = s.getNodeSelectors("foo", datastore.RequireCurrent)
+	selectors = s.getNodeSelectors("foo", datastore.TolerateStale)
```

Comment:
The order here seemed a bit more natural. First check that the function does what it's supposed to say and then check that the data is replicated.